fabric-biome-api-v1/src/main/java/net/fabricmc/fabric/impl/biome/NetherBiomeData.java

```diff
@@ -65,10 +76,43 @@ public static boolean canGenerateInNether(RegistryKey<Biome> biome) {
 			}
 		}
 
-		return NETHER_BIOMES.contains(biome);
+		return NETHER_BIOMES.contains(biome) || NETHER_BIOME_NOISE_POINTS.containsKey(biome);
 	}
 
 	private static void clearBiomeSourceCache() {
 		NETHER_BIOMES.clear(); // Clear cached biome source data
 	}
+
+	private static MultiNoiseUtil.Entries<RegistryEntry<Biome>> withModdedBiomeEntries(MultiNoiseUtil.Entries<RegistryEntry<Biome>> entries, Registry<Biome> biomeRegistry) {
+		if (NETHER_BIOME_NOISE_POINTS.isEmpty()) {
+			return entries;
+		}
+
+		ArrayList<Pair<MultiNoiseUtil.NoiseHypercube, RegistryEntry<Biome>>> entryList = new ArrayList<>(entries.getEntries());
+
+		for (Map.Entry<RegistryKey<Biome>, MultiNoiseUtil.NoiseHypercube> entry : NETHER_BIOME_NOISE_POINTS.entrySet()) {
+			if (biomeRegistry.contains(entry.getKey())) {
+				LOGGER.info("Nether biome {} loaded successfully", entry.getKey().getValue());
```

Comment:
Is it really necessary? I would not like unnecessary messages in my console

Reply:
> Also logging needs to be removed before we merge.